src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -7,55 +7,48 @@
     v-if="authorization"
     id="continuation-authorization"
   >
+    <!-- Authorization Date -->
     <v-card
       flat
       class="py-8 px-6"
       :class="{ 'invalid-section': getShowErrors && !authorizationDateValid }"
     >
-      <v-form
-        ref="formRef"
-        lazy-validation
-        @submit.prevent
-      >
```

Comment:
If you hide whitespace in this diff, you'll see that here I only  removed the unneeded form.